### PR TITLE
Feat: Card component

### PR DESCRIPTION
**Pull request by anuj-kumary**

### CODE REVIEW

**Review by Aman-Jain1996 — APPROVED** [review]
Looks good for merge

**Review by varnita619 — APPROVED** [review]
Everything seems perfect. Good to go 🚀

**Review by samyakshah3008 — APPROVED** [review]
### looks good to me!
Just one minor suggestion is to put comments in your code for better review experience, rest part is all good!

File: components/Card/card.css [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+  :root {
+    --bg-color: blueviolet;
+    --white: #fff;
+    --black: #111;
+  }
+  html {
+    font-size: 16px;
+  }
+  body {
+    font-family: sans-serif;
```
Comment (Parul1999) [inline components/Card/card.css]: Font-Family needs to be in CSS VARIABLE .

File: components/Card/card.css [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+  }
+  .footer__left p {
+    padding: 0 1rem;
+    color: var(--bg-color);
+  }
+  .footer__right {
+    margin: 0 0.9rem;
+    color: var(--black);
+  }
+  span {
```
Comment (Parul1999) [inline components/Card/card.css]: Styling on element tag is not to be done

**Review by Parul1999 — APPROVED** [review]

**Pull request merged**